### PR TITLE
Add tslib@1.6.0 w/ npm auto-update

### PR DESCRIPTION
**Pull request by lcd78706**
Pull request for issue: #10128 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/Microsoft/tslib
 * Official website (optional, not the repository):  http://typescriptlang.org/
 * NPM package url (optional): https://www.npmjs.com/package/tslib
 * License and its reference: Apache-2.0 [ref](https://github.com/Microsoft/tslib/blob/master/LICENSE.txt)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 70
   - Count of stars: 74
   - Count of forks: 6
 * NPM download stats (optional):
   - Downloads in the last day: 4,962
   - Downloads in the last week: 26,903
   - Downloads in the last month: 102,570

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by lcd78706** [comment]
@x09326 
Please review this PR. Thanks!

**Comment by lcd78706** [comment]
@x09326 
I have fixed it. Please review this PR again. Thanks!

File: ajax/libs/tslib/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "tslib",
+  "filename": "tslib.min.js",
+  "author": "Microsoft Corp.",
+  "homepage": "http://typescriptlang.org/",
```
Comment (pvnr0082t) [inline ajax/libs/tslib/package.json]: @lcd78706 Please use `https` instead.

Comment (lcd78706) [inline ajax/libs/tslib/package.json]: Cannot open the website after changing it to `https`.

Comment (pvnr0082t) [inline ajax/libs/tslib/package.json]: But I can access it. Is that your browser problem?

**Comment by lcd78706** [comment]
Wait for the author's response

File: ajax/libs/tslib/package.json (hunk 1 above)
Comment (weblucaseduardo) [inline ajax/libs/tslib/package.json]: hi, the link http://typescriptlang.org/ does not work with https for me, the link with www works fine with https, could we have that merged?

**Pull request closed without merging**